### DISCUSSION TITLE
Boundary Conditions on Inverse Problem

### DISCUSSION THREAD

**Discussion by gabrielord** [lululxvi/deepxde#1175] [Q&A]
Hello, Professor Lu Lu, I am sorry for bothering you.

I am trying to implement the following boundary condition:

![CodeCogsEqn (4)](https://user-images.githubusercontent.com/90040944/228622897-f96a2428-7ed2-4393-8daa-ec75e266cedc.gif)

Where phi and sigma are the output  and n^ is the outer vector at the point where the boundary condition is defined.

My difficult is on how to derivate phi with respect to this outer vector. 

I would appreciate if you could give any insight on how to implement this directional derivative.

**Comment by lululxvi** [comment]
I don't understand your question. Your problem is about boundary condition implementation, but why is it related to the inverse problem setup? You may remove those parts unrelated to your key question.

**Reply by gabrielord** [reply]
Thank you for your response and I'm sorry for my mistake.
I've updated my question.

**Comment by lululxvi** [comment]
Just compute it based on the definition.
